requirements.txt

```diff
@@ -2,3 +2,4 @@ PyGithub
 Waitress
 cryptography
 PyGHee>=0.0.3
+retry
```

Comment:
This looks useful, but it's also no longer actively developed (last update in 2016...)

As long as we don't experience problems with it, I guess that's OK, but we should wonder whether this is the best approach long-term.

How much work/effort would it be to just implement our own decorator, based on how `retry` is implemented?

Reply:
Maybe use it as is for now and create an issue to implement our own decorator? We could also check if there is any other package that does the same and is actively maintained. The repo has ~ 100 forks. We might also check if any of those is under active development?

Reply:
Yeah, ok to keep for now, just need to be aware that this is not actively maintained (it's likely going to be problematic on new Python 3.x versions at some point). So please open an issue

Reply:
Created issue #139